pkg/aws/ec2Client.go

```diff
@@ -164,3 +164,11 @@ func (e *EC2Client) GetManagedPrefixListEntries(ctx context.Context, params *ec2
 	}
 	return client.GetManagedPrefixListEntries(ctx, params, optFns...)
 }
+
+func (e *EC2Client) DescribeSubnets(ctx context.Context, params *ec2.DescribeSubnetsInput, optFns ...func(*ec2.Options)) (*ec2.DescribeSubnetsOutput, error) {
```

Comment:
@mnitchev are you changing this client in your refactoring? I don't see the point of having this wrapper around the sdk client at all.

Reply:
No I'm not. I'm creating a separate client for RAM

Reply:
We can change it in a future PR, it's fine for this PR.